src/module.h

```diff
@@ -24,6 +24,18 @@ extern "C" {
 
 #define PROXY_FILTERED "_proxy-filtered"
 
+// use likely and unlikely to provide the compiler with branch prediction information
+// for example:
```

Comment:
Maybe move this to a more general header?

Reply:
Like what?

Reply:
We can put it in its own file for explicitness